lib/active_support/cache/redis_store.rb

```diff
@@ -100,6 +100,8 @@ def read_multi(*names)
         return {} if names == []
 
         keys = names.map{|name| normalize_key(name, options)}
+        keys << options
```

Comment:
I don't like the readability here. It's OK to have a local var called `keys`, but I really don't think we should be shoveling in the `options` hash just to tack it on as another argument. Instead, how about keeping `keys`, but instead of shoveling in `options`, we do this?

```ruby
keys = keys = names.map{|name| normalize_key(name, options)}
args = [keys, options].flatten
values = with { |c| c.mget(*args) }
```